doc/changelogs/CHANGELOG_V13.md

```diff
@@ -36,6 +37,134 @@
   * [io.js](CHANGELOG_IOJS.md)
   * [Archive](CHANGELOG_ARCHIVE.md)
 
+<a id="13.7.0"></a>
+## 2020-01-21, Version 13.7.0 (Current), @codebytere
+
+### Notable Changes
+
+* **deps**:
+  * upgrade to libuv 1.34.1 (cjihrig) [#31332](https://github.com/nodejs/node/pull/31332)
+  * upgrade npm to 6.13.6 (Ruy Adorno) [#31304](https://github.com/nodejs/node/pull/31304)
+* **doc**:
```

Comment:
I suggest to make this an extra section below all others that says something like: `* **Added new collaborators**:`.